src/tests/roc_fec/target_openfec/test_writer_reader.cpp

```diff
@@ -898,7 +898,7 @@ TEST(writer_reader, repaired_block_numbering) {
     }
 }
 
-TEST(writer_reader, resize_source_block_begin) {
+TEST(writer_reader, writer_block_sizes) {
```

Comment:
As we do not use reader we can remove decoder as well.

Reply:
Done.